### PR TITLE
Core/Scripts: remove 2 unused scripts

### PR DESCRIPTION
**Pull request by ghost**
**Changes proposed:**

These scripts are no longer in use, may as well remove them.

**Target branch(es):** 3.3.5/6.x

**Issues addressed:** None in particular, just general cleanup.

**Tests performed:** (Does it build, tested in-game, etc) Builds without issues. Script removal only.

**Known issues and TODO list:**

None that I know about.

### CODE REVIEW

File: src/server/scripts/World/mob_generic_creature.cpp [hunk 1]
```diff
@@ -236,7 +79,6 @@ class trigger_death : public CreatureScript
 
 void AddSC_generic_creature()
 {
-    //new generic_creature;
     new trigger_periodic();
     //new trigger_death;
```
Comment (Shauren) [inline src/server/scripts/World/mob_generic_creature.cpp]: as you see here, trigger_death is also commented out (and unused)

Comment (tkrokli) [inline src/server/scripts/World/mob_generic_creature.cpp]: Thanks for confirming. :+1: Will fix.

File: src/server/scripts/World/world_script_loader.cpp [hunk 2]
```diff
@@ -41,7 +40,6 @@ void AddWorldScripts()
 {
     AddSC_areatrigger_scripts();
     AddSC_emerald_dragons();
-    AddSC_generic_creature();
```
Comment (Shauren) [inline src/server/scripts/World/world_script_loader.cpp]: now you went overboard with changes...

Comment (tkrokli) [inline src/server/scripts/World/world_script_loader.cpp]: Oh. right. That is only needed when removing the file. I will undo that.

**Pull request merged**